### PR TITLE
fix(v2): fix hash-link css

### PR DESCRIPTION
**Pull request by endiliey**
## Motivation

Currently, the hash-link is so hard to click.

See this gif or try it out at https://docusaurus-2.netlify.com/docs/installation/#launching-the-server-behind-a-proxy

Before I can click the hash, it already disappears
so hard to click

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Before
have to do it really fast

After

easier

## Related PRs

#1385

### CODE REVIEW

File: packages/docusaurus/lib/default-theme/DocBody/styles.css [hunk 1]
```diff
@@ -21,14 +21,16 @@
   position: absolute;
   margin-left: -1.25rem;
   padding-right: 0.25rem;
+  opacity: 0;
+  transition: opacity 0.3s;
 }
 
-@media only screen and (min-width: 736px) {
```
Comment (yangshun) [inline packages/docusaurus/lib/default-theme/DocBody/styles.css]: Why is this removed? This is for the hover behavior only on desktop widths since there's no hover on mobile.

Comment (endiliey) [reply packages/docusaurus/lib/default-theme/DocBody/styles.css]: dank that was true :| i forget hover doesnt work on mobile
